component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -311,11 +314,21 @@ protected void processAuthenticationResponse(HttpServletRequest request, HttpSer
                 throw new AuthenticationFailedException(errorMessage);
             }
             context.setSubject(authenticatedUser);
+            String emailFromProfile = context.getProperty(EmailOTPAuthenticatorConstants.RECEIVER_EMAIL).toString();
+            context.setSubject(
+                    AuthenticatedUser.createFederateAuthenticatedUserFromSubjectIdentifier(emailFromProfile));
         } else {
-            if (log.isDebugEnabled()) {
-                log.debug("Given otp code is mismatch");
+            if (isExpired) {
+                if (log.isDebugEnabled()) {
+                    log.debug("Given otp code is  expired.");
```

Comment:
Format the line remove the extra space in the log message?

Reply:
Fixed https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/65